src/Generator/Driver.cs

```diff
@@ -53,9 +53,6 @@ void ValidateOptions()
             {
                 if (string.IsNullOrWhiteSpace(module.LibraryName))
                     throw new InvalidOptionException("One of your modules has no library name.");
-
-                if (module.OutputNamespace == null)
-                    module.OutputNamespace = module.LibraryName;
```

Comment:
The problem here is that our API is not intuitive enough. If you set the output name-space to the empty string, you won't get a repetitive name-space. In other words, we make a difference between null and empty in this case but we haven't documented it anywhere. And this code is actually important because it automatically provides the user with a decent name-space. So for now please use the empty string instead and report an issue about this. I'll think of a way to improve our API.